x/ccv/provider/keeper/genesis_test.go

```diff
@@ -141,6 +153,17 @@ func TestInitAndExportGenesis(t *testing.T) {
 			ctx).Return(math.NewInt(100), nil).Times(1), // Return total voting power as 100
 	)
 
+	mocks.MockStakingKeeper.EXPECT().GetBondedValidatorsByPower(gomock.Any()).Return(
+		// return an empty set of validators, since they are not tested against here
```

Comment:
Why "empty set"? It seems here you return one validator (?) or maybe I misunderstood this line.

Reply:
Thanks, that was outdated from experimenting :(